mls-provider/Cargo.toml

```diff
@@ -44,6 +45,7 @@ zeroize = "1.5"
 thiserror = "1.0"
 hex = "0.4"
 async-lock = "3.3"
+generic-array = "0.14"
```

Comment:
Not needed as well, we can use any reexports of `generic_array` from our other crypto libraries (like signature, aead, ecdsa, etc)